src/sentry/static/sentry/app/components/events/contexts/trace/getTraceKnownDataDetails.tsx

```diff
@@ -0,0 +1,152 @@
+import React from 'react';
+import styled from '@emotion/styled';
+import moment from 'moment-timezone';
+
+import {Event, Organization} from 'app/types';
+import {t} from 'app/locale';
+import Button from 'app/components/button';
+import space from 'app/styles/space';
+import EventView from 'app/utils/discover/eventView';
+import {getTraceDateTimeRange} from 'app/components/events/interfaces/spans/utils';
+import {transactionSummaryRouteWithQuery} from 'app/views/performance/transactionSummary/utils';
+
+import {TraceKnownData, TraceKnownDataType} from './types';
+
+type Output = {
+  subject: string;
+  value: React.ReactNode | string | null;
+};
+
+function getUserKnownDataDetails(
+  data: TraceKnownData,
+  type: TraceKnownDataType,
+  event: Event,
+  organization: Organization
+): Output | undefined {
+  switch (type) {
+    case TraceKnownDataType.TRACE_ID: {
+      const traceId = data.trace_id || '';
+
+      if (!traceId) {
+        return undefined;
+      }
+
+      const dateCreated = moment(event.dateCreated).valueOf() / 1000;
+      const pointInTime = event?.dateReceived
+        ? moment(event.dateReceived).valueOf() / 1000
+        : dateCreated;
+
+      const {start, end} = getTraceDateTimeRange({start: pointInTime, end: pointInTime});
+
+      const orgFeatures = new Set(organization.features);
+
+      const traceEventView = EventView.fromSavedQuery({
+        id: undefined,
+        name: `Events with Trace ID ${traceId}`,
+        fields: ['title', 'event.type', 'project', 'trace.span', 'timestamp'],
+        orderby: '-timestamp',
+        query: `trace:${traceId}`,
+        projects: orgFeatures.has('global-views') ? [] : [Number(event.projectID)],
+        version: 2,
+        start,
+        end,
+      });
+      return {
+        subject: t('trace_id'),
+        value: (
+          <ButtonWrapper>
```

Comment:
I think you need a feature gate on this as am1 developer plans will have performance views but not discover2.

Reply:
should be addressed in https://github.com/getsentry/sentry/pull/19078/commits/5946793a0adc2c8e91fe45630a7219ce5251f8bc